.codeclimate.yml

```diff
@@ -8,6 +8,16 @@ engines:
     checks:
       overqualified-elements:
         enabled: false
+      order-alphabetical:
+        enabled: false
+      adjoining-classes:
+        enabled: false
+      fallback-colors:
```

Comment:
Not sure we need this for compatability.
`color: #fff;
color: rgba(255,255,255,.6)`